File: Java/app/src/main/java/software/rsquared/template/ui/main/MainPortraitNavigationController.java
package software.rsquared.template.ui.main;

import android.support.v7.app.AppCompatActivity;

import software.rsquared.template.utils.navigation.PortraitNavigationController;
import software.rsquared.template.utils.navigation.StartActivityBuilder;

public class MainPortraitNavigationController extends PortraitNavigationController implements MainNavigationController{


    public MainPortraitNavigationController(AppCompatActivity activity) {
        super(activity);
    }

    @Override
    public boolean goBack() {
        return false;
    }

    @Override
    public StartActivityBuilder navigateToSimpleExample() {
        return null;
    }
}

File: app/src/main/java/software/rsquared/template/ui/main/navigation/MainPortraitNavigationController.kt
package software.rsquared.template.ui.main.navigation

import software.rsquared.template.ui.main.MainActivity
import software.rsquared.template.utils.navigation.BaseNavigationController

class MainPortraitNavigationController constructor(activity: MainActivity) : BaseNavigationController(activity), MainNavigationController {

    override fun goBack(): Boolean {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun cleanup() {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }
}